.github/workflows/auto-assign.yml

```
@@ -0,0 +1,17 @@
+# Workflow for automatically assigning issues
+name: Issue auto-assign
+ on:
```

actionlint (1.7.10)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)